src/ansys/geometry/core/designer/body.py

```diff
@@ -868,6 +868,33 @@ def combine_merge(self, other: Union["Body", list["Body"]]) -> None:
         """
         return
 
+    def _combine_subtract(
+        self,
+        other: Union["Body", Iterable["Body"]],
+        keep_other: bool = False,
+        transfer_named_selections=True,
+    ) -> None:
+        """Subtract bodies from this body.
+
+        Warning: This is a specialized boolean operation that has the ability to transfer named
+        selections. It may behave differently than the encouraged ``subtract()``.
```

Comment:
```suggestion
        Warnings
        --------
        This is a specialized boolean operation that has the ability to transfer named
        selections. It may behave differently than the encouraged ``subtract()``.
```